src/discord/interaction_handlers/application_commands.rs

```diff
@@ -616,7 +657,7 @@ fn format_player(content: Option<String>, player: &Player) -> InteractionRespons
         fields,
         footer: None,
         image: None,
-        kind: "".to_string(),
+        kind: "rich".to_string(),
```

Comment:
the value here doesn't appear to be respected at all, but it seems better to pass the blessed value